python/sdist/setup.py

```diff
@@ -136,21 +150,12 @@ def main():
             'amici/libs',  # clib target directory
         ],
         extra_compile_args=['-std=c++11', *cxx_flags],
+        extra_compile_args_mingw32=['-std=c++11'],
+        extra_compile_args_unix=['-std=c++11'],
+        extra_compile_args_msvc=['/std:c++17'],
```

Comment:
could we stick with c++11 for msvc for the matter of consistency?

Reply:
See comment above. We cannot use c++11, but why do we need it? That would be a 8-year-old version of the standard.

Reply:
i would propose going for c++14 everywhere

Reply:
8-years old, but still not necessarily fully supported by every compiler...

Reply:
c++14 everywhere is fine with me.

Reply:
See earlier comment. Everything is now c++14.